homeassistant/components/backup/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""The Backup integration."""
+from homeassistant.components.hassio import is_hassio
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN, LOGGER
+from .http import async_register_http_views
+from .manager import BackupManager
+from .websocket import async_register_websocket_handlers
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the Backup integration."""
+    if is_hassio(hass):
+        LOGGER.warning(
```

Comment:
This is an error as the integration won't set up.
```suggestion
        LOGGER.error(
```